test/mocks/server/options.h

```diff
@@ -64,6 +65,7 @@ class MockOptions : public Options {
   bool allow_unknown_static_fields_{};
   bool reject_unknown_dynamic_fields_{};
   bool ignore_unknown_dynamic_fields_{};
+  bool suppress_deprecated_warning_logs_{};
```

Comment:
I don't think this is being used, and can probably be removed (the default return value of the mock can be false, and a test can override it if needed).

Reply:
removed this variable and directly updated the mock to return `false`.